### PR TITLE
chore(docs): update documentation `callbacks` to include user Id as example

### LINKED ISSUE #4903: Include `user.id` as a default value in session
### Description 📓

I have at least 3 or 4 users a week coming into the T3 Discord to ask specifically about getting a "user's ID from the `session` object"

I've copy-pasted the [snippet detailed in this issue so many times I've lost count](https://github.com/t3-oss/create-t3-app/issues/176). We're thinking of adding this by default to create-t3-app to help smooth out adoption of NextAuth

### How to reproduce ☕️

https://github.com/t3-oss/create-t3-app/issues/176

### Contributing 🙌🏽

Yes, I am willing to help implement this feature in a PR

### PR DESCRIPTION
**Pull request by ArkShocer**
## ☕️ Reasoning

Some people (including me) have been asking on how to add the userId to the client session. I think changing the documentation a bit to include the id would probably help a lot of people.

## 🧢 Checklist

- [x] Documentation
- [ ] Tests
- [ ] Ready to be merged

fixes #4903

### CODE REVIEW

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
I don't think this is the correct way to document this without explaining why it is not exposed by default. See https://github.com/nextauthjs/next-auth/issues/4903

**Comment by ArkShocer** [comment]
Notes on why the user id is not exposed by default is already documented in the `session` section. Are the latest changes in the correct direction or would it maybe make sense to move this to the faq since it's frequently asked?

File: docs/docs/configuration/callbacks.md [hunk 1]
```diff
@@ -112,15 +112,16 @@ Requests to `/api/auth/signin`, `/api/auth/session` and calls to `getSession()`,
 - As with database persisted session expiry times, token expiry time is extended whenever a session is active.
 - The arguments _user_, _account_, _profile_ and _isNewUser_ are only passed the first time this callback is called on a new session, after the user signs in. In subsequent calls, only `token` will be available.
 
-The contents _user_, _account_, _profile_ and _isNewUser_ will vary depending on the provider and on if you are using a database or not. You can persist data such as User ID, OAuth Access Token in this token. To make it available in the browser, check out the [`session()` callback](#session-callback) as well.
+The contents _user_, _account_, _profile_ and _isNewUser_ will vary depending on the provider and on if you are using a database or not. You can persist data such as User ID, OAuth Access Token in this token, see example below for `Access Token` and `User ID`. To make it available in the browser, check out the [`session()` callback](#session-callback) as well.
```
Comment (balazsorban44) [inline docs/docs/configuration/callbacks.md:115]:
```suggestion
The contents _user_, _account_, _profile_ and _isNewUser_ will vary depending on the provider and if you are using a database. You can persist data such as User ID, OAuth Access Token in this token, see the example below for `access_token` and `user.id`. To expose it on the client side, check out the [`session()` callback](#session-callback) as well.
```

File: docs/docs/configuration/callbacks.md [hunk 2]
```diff
@@ -134,7 +135,7 @@ Use an if branch to check for the existence of parameters (apart from `token`).
 
 ## Session callback
 
-The session callback is called whenever a session is checked. By default, only a subset of the token is returned for increased security. If you want to make something available you added to the token through the `jwt()` callback, you have to explicitly forward it here to make it available to the client.
+The session callback is called whenever a session is checked. By default, **only a subset of the token is returned for increased security**. If you want to make something available you added to the token (like `Access Token` and `User ID` from above) through the `jwt()` callback, you have to explicitly forward it here to make it available to the client.
```
Comment (balazsorban44) [inline docs/docs/configuration/callbacks.md:138]:
```suggestion
The session callback is called whenever a session is checked. By default, **only a subset of the token is returned for increased security**. If you want to make something available you added to the token (like `access_token` and `user.id` from above) via the `jwt()` callback, you have to explicitly forward it here to make it available to the client.
```

**Review by balazsorban44 — APPROVED** [review]
Thank you!

**Pull request merged**